README.md

```diff
@@ -59,28 +59,50 @@ Options:
   -w, --write [value]                     file path to write results
   -e, --excluded [value]                  The comma separated list of tests to exclude (default: none)
   -x, --externalPluginsDirectory [value]  Relative or full path to an external plugins directory
+  -q, --quiet                             do not emit the report to stdout. (can use --write option to write to file)
   --list                                  do not execute, instead list the available plugins and formatters
   --maxWarnings [value]                   Number of warnings to trigger nonzero exit code (default: -1)
   --profile [value]                       Either apigee or apigeex (default: apigee)
   -h, --help                              output usage information
 ```
 Example:
 ```
-apigeelint -s sampleProxy/ -f table.js
+apigeelint -s sampleProxy/apiproxy -f table.js
 ```
 
 Where `-s` points to the apiProxy source directory and `-f` is the output formatter desired.
 
 Possible formatters are: "json.js" (the default), "stylish.js", "compact.js", "codeframe.js", "html.js", "table.js", "unix.js", "visualstudio.js", "checkstyle.js", "jslint-xml.js", "junit.js" and "tap.js".
 
-Example Using External Plugins:
+### More Examples
+
+#### Using External Plugins:
+```
+apigeelint -x ./externalPlugins -s path/to/your/apiproxy -f table.js
+```
+Where `-x` points to the directory containing externally developed plugins.
```

Comment:
Probably mention what `-e` is
You removed a line that had the description of what `-e` is

Reply:
done